.github/workflows/crucible-ci.yaml

```diff
@@ -186,7 +186,7 @@ jobs:
       matrix:
         endpoint: [ "k8s" ]
         benchmark: [ "oslat" ]
-        userenv: [ "default" ]
+        userenv: [ "rhubi8" ]
```

Comment:
We need to drop this change.  This directly affects how much testing is being done because the `default` userenv undergoes additional testing that the other other userenvs do not.